### ISSUE TITLE
no matter what i do i cant create the Metasloitable3

### ISSUE THREAD

**Issue by caloss1** [rapid7/metasploitable3#275]
## Issue Description
**Please check the [General Issues](https://github.com/rapid7/metasploitable3/wiki/General-Issues) 
*i follow the all YouTube tutorial and start the powershell script, the installation stuck every time by error

i try different version of packer and vagrant, nothing its seems the entire setup is corrupted  

## Host System
* OS: Windows 10
* Packer Version: 1.1.1 (i try with the newest
![meta](https://user-images.githubusercontent.com/38428948/38812183-42808fca-4194-11e8-82fc-4581eacef89f.png)
 version , error wont stop)
* Vagrant Version: 2.0.3 
* VirtualBox Version: 5.2.8

## Command Output
Copy the relevant command output here.
If it's long, either post to a gist and add the link here, or isolate the error lines.

**Comment by amdorj** [comment]
Try downloading Vagrant 1.9.1 and running it with that. According to the General Issues page Vagrant 1.9.1 is the currently supported version.

**Comment by mkienow-r7** [comment]
The (General Issues)[https://github.com/rapid7/metasploitable3/wiki/General-Issues] wiki page looks like it hasn't been updated recently. Where is the General Issues page? @caloss1 The screenshot doesn't show all of the output. What command did you run? Have you tried using packer version 1.2.2? I haven't worked with the PS build, but some of the paths looks a bit odd to me with the mixed path separators `/` and `\`. Your screenshot doesn't show all of the console output. What command did you run that produced the above output?

**Comment by caloss1** [comment]
i install excact like the youtbue toturial 
the command i use is 
 .\build_win2008.ps1

**Comment by caloss1** [comment]
still dont work, what i can do?

**Comment by jmartin-r7** [comment]
According to your output the box built properly it just need to be added to vagrant

I would try moving into the folder where the box is located and import there.

```
cd packer\builds\
vagrant box add metasploitable3-win2k8 windows_2008_r2_virtualbox.box
```

**Comment by dukeofdisaster** [comment]
clone from this user's fork https://github.com/Chan9390/metasploitable3 and build.... make sure you remove  any existing vagrant boxes prior 

PS> vagrant box list

then delete any of the listed box with 
PS> vagrant box remove <insertBoxNameHere>

His build script worked for me on Windows 10 Pro

**Comment by Chan9390** [comment]
Hey @caloss1 , sorry for late reply.

From the logs I could notice a box is created at `J:\DownloadGames\metasploitable3-master\packer\builds\windows_2008_r2_virtualbox_0.1.0.box`. But when the script adds the box, it is looking for the box file `windows_2008_r2_virtualbox.box` which doesn't exist.

**Solution:**
```
cd packer\builds\
vagrant box add metasploitable3-win2k8 windows_2008_r2_virtualbox_0.1.0.box
```

**Comment by Chan9390** [comment]
Merging PR #277 will resolve this issue.

**Comment by jmartin-r7** [comment]
Closing as fixed with #277
